### DISCUSSION TITLE
How can I configure automatic recording based on *time*

### DISCUSSION THREAD

**Discussion by KeithBierman** [roflcoopter/viseron#658] [Q&A]
The goal is record each class, so a schedule like M-F 8:00-8:855, 9:00-9:55, etc would be ideal. However, even just an automatic start at a given time, followed by a duration (or stop time) for each day would suffice. How might one configure this?

**Comment by roflcoopter** [comment]
This is not possible at the moment, but it is on my radar.
You can stop and start a camera using MQTT tho, effectively arming / disarming the camera. This requires an external tool tho like Home Assistant
